config/database.js

```diff
@@ -1,3 +1,22 @@
+// MAKE SURE TO ONLY USE ONE 
+
+const mongoose = require('mongoose');
+
+mongoose.connect(process.env.DATABASE_URL, {
+    useNewUrlParser: true, 
+    useCreateIndex: true,
+    useUnifiedTopology: true 
+});
+
+const db = mongoose.connection
+db.once('open', _ => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7821591)
```suggestion
  console.log("Database connected:", process.env.DATABASE_URL)
```